src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,75 @@
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: React.Dispatch<number>;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+  const isFirstPage = currentPage === 1;
+  const isLastPage = currentPage === totalPages;
+
+  const handlePageChange = (page: number) => {
+    if (page !== currentPage) {
+      onPageChange(page);
+    }
+  };
+
+  return (
+    <div>
+      <ul className="pagination">
+        <li className={`page-item ${isFirstPage ? 'disabled' : ''}`}>
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={isFirstPage}
+            onClick={() => !isFirstPage && handlePageChange(currentPage - 1)}
```

Comment:
The 'onClick' handler here can invoke 'handlePageChange' directly without the need for the conditional check since 'handlePageChange' already performs this check.